docs/technical_reference/unit_models/pump.rst

```diff
@@ -79,10 +79,18 @@ then the pump efficiency is assumed to depend only on flow:
          0.4 & \text{for } r_{bp, t} > 1.4
        \end{cases}
 
+**NOTE: this option for centrifugal pumps has been used in modeling but not validated for high pressure applications**
```

Comment:
Can you add the reference like so:


When the configuration ``variable_efficiency`` is set to ``VariableEfficiency.flow``,
then the pump efficiency is assumed to depend only on flow (Kuritza et al., 2017):

Reply:
Revised accordingly